Assets/Scripts/Network/ColyseusClient.cs

```diff
@@ -231,19 +231,19 @@ public async void SendNetworkTransformUpdate(Vector3 pos, Quaternion rot, Vector
         }
     }
 
-    public async void SendNetworkAnnotationUpdate(Vector3 startPos, Vector3 endPos, string transformName, NetworkMessageType messageType)
+    public async void SendNetworkAnnotationUpdate(Vector3 startPos, Vector3 endPos, Vector3 rotation, string transformName, NetworkMessageType messageType)
     {
         if (IsConnected)
         {
             NetworkTransform t = new NetworkTransform();
             // This is a hack to use the NetworkTransform class to communicate the start/end positions
             // of the annotation. The NetworkTransform class is designed to store center, scale, and rotation,
-            // but for now we will store the start and end positions of the annotation in the first two vector3 slots.
+            // but for now we will store the start and end positions of the annotation in the first and third vector3 slots.

```

Comment:
See my previous comment on this.  It seems like the start and end are still in the first two positions.

Reply:
I updated the comment to clarify what is happening here.  Sorry for the confusion.